config/jobs/kubernetes-csi/external-attacher/external-attacher-config.yaml

```diff
@@ -3,10 +3,7 @@
 presubmits:
   kubernetes-csi/external-attacher:
   - name: pull-kubernetes-csi-external-attacher-1-13-on-kubernetes-1-13
-    # Experimental job, explicitly needs to be started with /test.
-    # This can be enabled once the components have the necessary configuration
-    # for this combination of deployment+Kubernetes.
-    always_run: false
+    always_run: true
     decorate: true
     skip_report: false
     skip_branches: ["^(release-0.2.0|release-0.3.0|release-0.4|release-1.0|saad-ali-patch-1|saad-ali-patch-2|saad-ali-patch-3|v0.1.0)$"]
```

Comment:
I went through all the sidecar repos and removed all the "saad*" branches, so this can be cleaned up a little too